doc/source/whatsnew/v0.25.0.rst

```diff
@@ -332,6 +332,7 @@ Timezones
 - Bug in :func:`to_datetime` where an uninformative ``RuntimeError`` was raised when passing a naive :class:`Timestamp` with datetime strings with mixed UTC offsets (:issue:`25978`)
 - Bug in :func:`to_datetime` with ``unit='ns'`` would drop timezone information from the parsed argument (:issue:`26168`)
 - Bug in :func:`DataFrame.join` where joining a timezone aware index with a timezone aware column would result in a column of ``NaN`` (:issue:`26335`)
+- FutureWarning message raised by :func:`DatetimeIndex.to_series` updated to identify source of warning (:issue:`26329`)
```

Comment:
don't need a note for this

Reply:
Meta-question: with review requests like this, is it better to just merge? I also thought that the note was unnecessary, but the "harm" of this note is small. Maybe better to just merge, and reduce CI burden / risk of a merge conflict coming up?

Reply:
i don’t think the ci burden is relevant here
we are pretty fast now adays
and better to fix things even small typos on the first go
as once things are merged it is almost impossible to see that something is amiss